### DISCUSSION TITLE
Question about my IT i7+3 Pool Config not sticking

### DISCUSSION THREAD

**Discussion by sdpools** [tagyoureit/nodejs-poolController#407] [RS485]
Hello,

I bought a intellitouch used for a few hundred dollars. I don't have a ICP. I thought PoolController would be a good way to program the unit. Pi is outside in loadcenter. I have a direct connect usb FTDI RS-485 connected. I also have a RS-485 to ethernet /VCOM controller connected. Both are configured in the dash - both pass data. I got the old owners config to come up, but was having issues modifying the config via the web dash to reflect my build. I then reset the OCP via reset button->#5. This cleared out the old stuff. I also rebuilt the server a few times due to me learning linux.

Issue is: I am having a hard time keeping many of my web configurations to stick.
Specifically: Valves and Features.
I see the command for PUT across the controller CLI (silly logging), it says "saving", then times out it seems like.

Every once in a while it will take, and then I bang my head through the next one.
I know I can build it in data/PoolConfig.json, Is this the only solution?

A few questions:
1. the Intellitouch i7+3 is recognized, however, the firmware version is not displayed (undefined)? Is that normal?
2. What does this value mean? "intakeReturnValves": false, - I have intake, valve A, valve B, but return is not shown under valves?

. "maxPumps": 8,
    "maxValves": 4,
    "maxCircuitGroups": 10,
    "maxChlorinators": 1,
    "maxChemControllers": 1,
    "maxCustomNames": 20,
    "maxFeatures": 8,
    "model": "IntelliTouch i7+3",
    "modules": [
      {
        "id": 0,
        "master": 0,
        "name": "IT7",
        "desc": "IntelliTouch i7+3",
        "type": 1,
        "part": "i7+3",
        "bodies": 2,
        "circuits": 8,
        "features": 8,
        "valves": 4,
        "maxPumps": 8,
        "intakeReturnValves": false,
        "chlorinators": 1,
        "chemControllers": 1
      }
    ],
    "shared": true,
    "dual": false,
    "intakeReturnValves": false,
    "expansions": [],

**Comment by rstrouse** [comment]
We are going to need some replay information for this.  If you can submit this as an issue and include the replay information we can why the Custom names are getting blown away and why the solar settings are not sticking.  

We just set this discussions section up so I think there is a Create issue from discussion tool but we need the replay data to look at the comms and responses from the panel.

https://github.com/tagyoureit/nodejs-poolController/wiki/How-to-capture-all-packets-for-issue-resolution

**Comment by sdpools** [comment]
rstrouse,

I will move over to create issue forum.

Thank you

**Comment by tagyoureit** [comment]
Not sure if this is available to everyone (hope it is), but the create issue button is in the RH toolbar.  

![image](https://user-images.githubusercontent.com/1646840/162672501-ee8b2a18-5847-4548-a0ad-2262a8ef07c1.png)

Ref: https://github.com/tagyoureit/nodejs-poolController/issues/410

**Reply by sdpools** [reply]
![49A19797-6AD6-4A43-9C48-1734A2A77D2A](https://user-images.githubusercontent.com/103350476/162884524-734d61d5-64af-4c4a-9432-8ee15507d2a5.jpeg)

I found the links on the mobile format also (IOS). Located on the very bottom of the thread / page.

I didn’t transfer this topic because I already reposted a version of the question. 

Cool feature though.

**Comment by flyboyusa** [comment]
Subscribed....I'm having a very similar issue on my panel.  My changes are not sticking whenever the njsPC reloads the panel config

**Reply by tagyoureit** [reply]
Can you see the previous posts and upload a replay capture so we can see what is going on?

NVM - I see you added one to #560. Ty!
